models/bamf_nnunet_ct_kidney/config/default.yml

```diff
@@ -0,0 +1,37 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: default configuration for Bamf NNUnet Kidney segmentation (dicom to dicom)
+
+execute:
+- DicomImporter
+- NiftiConverter
+- NNUnetRunner
+- BamfProcessorRunner
+- DsegConverter
+- DataOrganizer
+
+modules:
+  DicomImporter:
+    source_dir: input_data
+    import_dir: sorted_data
+    sort_data: true
+    meta: 
+      mod: '%Modality'
+
+  NNUnetRunner:
+    in_data: nifti:mod=ct
+    nnunet_task: Task779_Kidneys_KIRC
+    nnunet_model: 3d_fullres
+    roi: LEFT_KIDNEY, RIGHT_KIDNEY
```

Comment:
Spaces might cause a failure here `, `. 
If RIGHT_KIDNEY is not found, or you encounter an error, try removing the leading space as it might be interpreted as part of the keyword (we currently don't do any white space stripping, but I've already included it for a future release). 

Please let me know if this caused an error during testing! :)